themes/Metro.theme.xml

```diff
@@ -594,20 +594,15 @@
                                 <VisualState x:Name="Unfocused"/>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                        <TextBlock x:Name="UnderlineTextBlock"
-                                   HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                                   Margin="{TemplateBinding Padding}" Text="{TemplateBinding Content}"
-                                   TextDecorations="Underline" Visibility="Collapsed"
-                                   VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
-                            <TextBlock.Foreground>
+                       
+                        <Border x:Name="UnderlineTextBlock" Margin="2,0,2,1" BorderThickness="0,0,0,1" Visibility="Collapsed">

```

Comment:
I can try it and see what happens - the offshore team has already asked about the underline that shows up. This way, it can be controlled by template. I like this better